relayer/spy_relayer/src/listener/validation.ts

```diff
@@ -58,7 +62,7 @@ export function validateInit(): boolean {
 
 export async function parseAndValidateVaa(
   rawVaa: Uint8Array
-): Promise<string | ParsedVaa<ParsedTransferPayload>> {
+): Promise<string | ParsedVaa<ParsedTransferPayload> | ParsedVaa<ParsedTransferWithArbDataPayload<ParsedSwimData>>> {
```

Comment:
not relaying payloadType === 1 so don't need to allow returning a `ParsedVaa<ParsedTransferPayload>`